contracts/HashRegistrarSimplified.sol

```diff
@@ -381,7 +416,7 @@ contract Registrar {
         Deed bid = sealedBids[bidder][seal];
         // If the bid hasn't been revealed after any possible auction date, then close it
         if (address(bid) == 0
-            || now < bid.creationDate() + initialAuctionPeriod) throw;
+            || now < bid.creationDate() + launchLength) throw;
```

Comment:
If we want to give people a grace period, then, we should explicitly set that grace period - there's no reason it needs to be identical to the soft launch period.